frontend/awx/resources/templates/TemplatesList.cy.tsx

```diff
@@ -0,0 +1,138 @@
+import { TemplatesList } from './TemplatesList';
+import * as useOptions from '../../../common/crud/useOptions';
+import { ToolbarFilterType } from '../../../../framework';
+
+describe('TemplatesList', () => {
+  describe('ErrorList', () => {
+    it('displays error if templates list is not successfully loaded', () => {
+      cy.intercept({ method: 'GET', url: '/api/v2/projects/6/*' }, { statusCode: 500 });
+      cy.mount(<TemplatesList url={'/api/v2/projects/6/'} />);
+      cy.contains('Error loading templates');
+    });
+  });
+
+  describe('Non-empty list', () => {
```

Comment:
@daphnemaeve is there a test that covers empty state error message when there are no templates to display?

Reply:
There is a test to display an error if the list is not correctly loaded, but I can add one to test the behavior if the list loads but has no templates

Reply:
@vishsanghishetty Added tests for empty list! Please let me know if they look good to you